.circleci/config.yml

```diff
@@ -1027,6 +1034,7 @@ workflows:
                     - build_ig_test_cms_web
                     - build_malte_test_cms_web
                     - build_aschaffenburg_web
```

Comment:
This won't work, you have to add it as a step to the `if development` in `deliver_web`